packages/tomls/src/exchange_pass_nft/configs/mint.toml

```diff
@@ -7,3 +7,23 @@ args = [
     "<%= solidityKeccak256(['address','uint256'],[settings.passivePoolExchangeFeeCollectorOwner, 1]) %>",
 ]
 depends = ["invoke.upgrade_exchange_pass_proxy"]
+
+[invoke.exchange_pass_lottery_pot_mint_owner]
```

Comment:
hmm, strange that this wasn't here already given we executed this already on both testnet and mainnet